lighthouse-core/scripts/open-devtools.sh

```diff
@@ -19,28 +24,13 @@ if [ -z "${CHROME_PATH:-}" ]; then
   exit 1
 fi
 
-echo "CHROME_PATH: $CHROME_PATH"
-echo "DEVTOOLS_PATH: $DEVTOOLS_PATH"
-
-if [ ! -d "$DEVTOOLS_PATH" ]; then
-  echo "No devtools found at $DEVTOOLS_PATH. Have you run 'yarn test-devtools' yet?"
-  exit 1
+if [ "$DEFAULT_DEVTOOLS_PATH" == "$DEVTOOLS_PATH" ]; then
+  bash "$LH_ROOT/lighthouse-core/test/chromium-web-tests/setup.sh"
 fi
 
-if ! which gn ; then
-  # If the contributor doesn't have a separate depot tools in their path, use the tmp copy.
-  DEPOT_TOOLS_PATH="$TEST_DIR/depot-tools"
-  BLINK_TOOLS_PATH="$TEST_DIR/blink_tools"
-  export PATH=$DEPOT_TOOLS_PATH:$PATH
-  # Add typ to python path. The regular method assumes there is a Chromium checkout.
-  export PYTHONPATH="${PYTHONPATH:-}:$BLINK_TOOLS_PATH/latest/third_party/typ"
-fi
-
-yarn devtools "$DEVTOOLS_PATH"
+echo "CHROME_PATH: $CHROME_PATH"
 
-cd "$DEVTOOLS_PATH"
-gn gen out/Default
-gclient sync
-autoninja -C out/Default
+export DEVTOOLS_PATH
```

Comment:
nit: could we just add `export` to L20?

Reply:
I think having it here makes it clear that the purpose is for the following script to have it defined.

Reply:
ah but with the refactor you suggested it can be removed